src/Conference/ConferenceDetails.test.js

```diff
@@ -48,20 +52,19 @@ describe("ConferenceDetails", () => {
     }
   };
 
-  it("renders conference details", (done) => {
+  it("renders conference details", done => {
     expect.assertions(1);
 
     axios.get.mockResolvedValue(mockData);
 
-    useParams.mockReturnValue({ id: '1' });
+    useParams.mockReturnValue({ id: "1" });
 
     const component = mount(<ConferenceDetails />);
 
     setImmediate(() => {
       component.update();
       expect(component.debug()).toMatchSnapshot();
```

Comment:
fyi I don't think you need to use `.debug()` anymore as you have a serialiser in your package.json :)